packages/orbit-tailwind-preset/src/index.ts

```diff
@@ -8,21 +8,19 @@ import getComponentLevelTokens, { ExportedComponentLevelTokens } from "./getComp
 import cssVarsFoundation from "./foundation/cssVarsFoundation";
 
 const COLORS: Partial<ExportedComponentLevelTokens>[] = [
+  "alert",
+  "badge",
   "button",
   "buttonLink",
+  "countryFlag",
   "drawer",
-  "socialButton",
-  "alert",
-  "icon",
   "formElement",
-  "badge",
-  "tag",
+  "heading",
+  "icon",
   "tab",
-  "textLink",
+  "tag",
   "text",
-  "loading",
-  "heading",
-  "countryFlag",
+  "textLink",
```

Comment:
I wonder if this is a BC

Reply:
They're marked as BC on the individual commits that correspond to the class removals. This only tells the build phase to not consider those component tokens (because they no longer exist). Plus, this refers to component-specific classes, that are undocumented (although I documented their removal as BC 🤷‍♂️), so I don't think this step in particular needs to be a BC

Reply:
Oh true, this is actually not being exported.

I meant the change on [getComponentLevelTokens.ts](https://github.com/kiwicom/orbit/pull/4560/commits/b1ad36c627bc52630426d2094cf3c18ebd5062f6#diff-abb5f236013bdc7f6e61903f2493cfef57942671d9a7edf4e0220b8e4fa372a2), my bad.

In that case, an exported type is being altered. I understand it's internal and undocumented; I'm just trying to make sure this doesn't have side effects on consumers.